tsconfig.lib.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "./tsconfig.base.json",
+  "exclude": ["node_modules"],
+  "includes": ["**/*.ts", "**/*.test.ts"]
```

Comment:
If the definitions of lib and spec are the same, there is no need to separate them, right?

Reply:
woops that's a mistake. one is supposed to exclude tests the other include it.